src/main/java/org/jmeterplugins/repository/PluginManagerCMD.java

```diff
@@ -108,8 +108,9 @@ protected void installPluginsForJmx(ListIterator jmxFilesIterator) throws Throwa
         PluginManager mgr = getPluginsManager();
         PluginSuggester suggester = new PluginSuggester(mgr);
         final Set<Plugin> pluginsToInstall = new HashSet<>();
-        if (jmxFilesIterator.hasNext()) {
-            pluginsToInstall.addAll(suggester.analyzeTestPlan(jmxFilesIterator.next().toString()));
+        Set<String> jmxFiles = parseParams(jmxFilesIterator.next().toString()).keySet();
```

Comment:
What's the point of having sets and iterators? Does not look as clear behavior. Maybe you want to enable `tool.sh install-for-jmx file1.jmx file2.jmx`? 
ANyway, this piece with `next()` does not look right to me. Can you review and refacror this to be clearer?

Reply:
Yes, I want to enable list comma-separated files. I think this behavior is more like other commands